tests/time_entries/time_enties_namespace_test.py

```diff
@@ -0,0 +1,14 @@
+from flask import json
+
+def test_list_should_return_empty_array(mocker, client):
```

Comment:
Could we have type hints here?
Also, check the name of the file, I think It should be 'time_entries_namespace_test' or why not just 'time_entries_test' ?

Reply:
Ok